modules/node_modules/@ncigdc/containers/CancerDistributionTable.js

```diff
@@ -99,7 +98,7 @@ const CancerDistributionTableComponent = compose(
           key={b.key}
           ssms={explore.ssms}
           filters={makeFilter([
-            { field: 'genes.gene_id', value: [geneId] },
+            { field: 'genes.symbol', value: [entityName] },
```

Comment:
correcting that, "it's the two genes have the same symbol, the symbol is marked with a `*`"